src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/AuthorisedGetEventTriggerOperation.java

```diff
@@ -1,15 +1,10 @@
 package uk.gov.hmcts.ccd.domain.service.aggregated;
 
-import java.util.Set;
```

Comment:
when testing locally please make sure you don't have any [CREATOR] case role already assigned explicitly to your user in `case_users` table.